### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by ryan-c-dev** [carterprince/libby#3]
Would be nice if you could choose which program handles the view mode.

Use Zathura.

Otherwise great software, keep up the good work and thanks for handling my issue request from before so quickly!

**Comment by carterprince** [comment]
I was actually thinking about adding this.

I think the best way to do this is to read from an environment variable such as `LIBBY_VIEWER`. If the variable is empty, 
 libby would default to opening files with `xdg-open`. Otherwise, it will try to open the file with the specified command.

Does this sound like what you are looking for?

**Comment by carterprince** [comment]
This is now implemented in 7104cc6179223aebf4afab4c2c1c7fcf05a04066.

To change the command, set the `LIBBY_VIEWER` environment variable in your `.zprofile` / `.bash_profile`:

```sh
export LIBBY_VIEWER="foobar"
```

However, if `zathura` is your preferred document viewer (it's mine too), the simpler solution is to set it as your default viewer with `xdg-mime`:

```sh
xdg-mime default org.pwmt.zathura.desktop application/pdf
xdg-mime default org.pwmt.zathura.desktop application/epub+zip
```

That way, `xdg-open` will always open ePUBs and PDFs in `zathura`.
